browser.js

```diff
@@ -47,7 +47,10 @@ not, since the functions are different and does
 not convert using browserify */
 function randomBytes(size) {
   var arr = new Uint8Array(size);
-  if (typeof window === 'undefined') {
+  if (
+    typeof browserCrypto === 'undefined' ||
```

Comment:
have you tested this? I don't think it will ever be undefined because the fallback on line 7 is an empty object.